libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1270,19 +1282,25 @@ def test_deployment(self, sns_data, sns_parity,
             resp = self.deploy_cortx_cluster(sol_file_path, master_node_list,
                                              worker_node_list, system_disk_dict,
                                              self.git_script_tag, namespace)
-            assert_utils.assert_true(resp[0], resp[1])
-            LOGGER.info("Step to Check  ALL service status")
-            time.sleep(60)
-            service_status = self.check_service_status(master_node_list[0])
-            LOGGER.info("service resp is %s", service_status)
-            assert_utils.assert_true(service_status[0], service_status[1])
-            row.append(service_status[-1])
-            if self.deployment_type != self.deploy_cfg["deployment_type_data"]:
-                if self.cortx_server_image:
-                    resp = self.verfiy_installed_rpms(master_node_list,
-                                                      common_const.RGW_CONTAINER_NAME,
-                                                      self.deploy_cfg["rgw_rpm"])
-                    assert_utils.assert_true(resp[0], resp[1])
+            if len(namespace) >= 63 or bool(re.match(r'\w*[A-Z]\w*', namespace)):
```

Comment:
For positive scenario (continuous deployment), validate if length < 63 and all character are lower case.

Reply:
done